addons/binding/org.openhab.binding.yamahareceiver/ESH-INF/thing/thing-types.xml

```diff
@@ -64,7 +71,10 @@
 		<description>Control the zone</description>
 		<channels>
 			<channel id="power" typeId="power" />
+			<!--
+            We will provide an input channel dynamically. 
```

Comment:
@martinvw I didn't know this. Is there a blog post or something that elaborates on this? What's the alternative? Yamaha returns a different set of input channels back, depending on the features of the AVR. This can only be done dynamically.